include/nccl_ofi_param.h

```diff
@@ -164,22 +164,11 @@ OFI_NCCL_PARAM_INT(domain_per_thread, "DOMAIN_PER_THREAD", -1);
 
 /*
  * When enabled and RDMA communication protocol is used, write NCCL
- * topology file and set environment variable `NCCL_TOPO_FILE`. By
- * default, OFI plugin writes the NCCL topology file to a unique
- * temporary file using file path template
- * `/tmp/aws-ofi-nccl-topo-XXXXXX` and the file is deleted at normal
- * process termination. See environment variable
- * `OFI_NCCL_TOPO_FILE_TEMPLATE` to control the file destination.
+ * topology file and set environment variable `NCCL_TOPO_FILE`. OFI plugin
+ * writes the NCCL topology file to a memfd file.
  */
 OFI_NCCL_PARAM_INT(topo_file_write_enable, "TOPO_FILE_WRITE_ENABLE", 0);
```

Comment:
I don't think we should have this env var any more.  It made sense when there were downsides to writing the file and we were trying to move fast to get P5 shipped (and knew we were leaving turds around), but now we have an approach we like, and customers can always set `NCCL_TOPO_FILE=/dev/null` if they really need to.